src/components/BuySellForm/BuySellForm.tsx

```diff
@@ -177,26 +194,70 @@ const BuySellForm = ({ advertId, isModalOpen, onRequestClose }: TBuySellFormProp
         }
     };
 
+    const onCloseBuySellForm = () => {
+        onRequestClose();
+        unsubscribe();
+        invalidate('p2p_advert_list');
```

Comment:
why do we need to invalidate p2p_advert_list ? 🤔

Reply:
when the counterparty changes the rate and then you close that error modal, then close the Buy sell modal, the rate isn't updated there in the list, so this will make sure the data in the table is updated once the user closes the modal

Reply:
but shouldn't it be called from useordercreate onsuccess?

Reply:
not really, cause this is for when the user doesn't place any order, and just exits it once counterparty had changed the rate, when the user already places the order, itll go to orders page and when they come back to buy/sell page, it will already request for new data

Reply:
ah.. okay.. got it

Reply:
so actually the list will always show the stale data no matter how many changes were there.. right? until the user doesn't open the buy/sell form and close it 🤔

Reply:
yeah, unless we add a periodic refetch interval, but i think thats alright, we can experiment with it later